tests/core/test_open_in_process.py

```diff
@@ -51,6 +51,7 @@
             while True:
                 await asyncio.sleep(0)
```

Comment:
Interestingly tests also hang weirdly if you change this sleep to be `0.001` or other small non-zero values.